tools/test_functions.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
```

Comment:
On further inspection, I think you should combine all of you tests into a single file in `dagmc/tools/tests` called `test_uwuw_preproc`.